app/scss/pages/_home.scss

```diff
@@ -15,104 +16,306 @@
 			margin-bottom: 25px;
 		}
 		@media (max-width : 480px) {
 			font-size: 17px;
 		}
 	}
+	.room {
+		display: flex;
+		justify-content: space-between;
+		margin-bottom: 70px;
+		@media (max-width : 990px) {
+			flex-direction: column;
+			justify-content: center;
+		}
+		@media (max-width : 550px) {
+			margin-bottom: 50px;
+		}
+		@media (max-width : 480px) {
+			margin-bottom: 35px;
+		}
+		&__info {
+			display: flex;
+			flex-direction: column;
+			justify-content: center;
+			flex: 1;
+			margin-right: 70px;
+			@media (max-width : 990px) {
+				text-align: center;
+				margin-right: 0;
+				order: 2;
+				margin-top: 40px;
+			}
+			&-inner {
+				position: relative;
+				&::before {
+					content: "";
+					border: 1px solid #14274A;
+					position: absolute;
+					left: -35px;
+					height: 100%;
+					@media (max-width : 1170px) {
+						display: none;
+					}
+				}
+			}
+			&-title {
+				font-family: AGaramondPro, sans-serif;
+				font-size: 50px;
+				letter-spacing: 0.02em;
+				font-weight: 700;
+				letter-spacing: 0.02em;
+				line-height: 1.1;
+				@media (max-width : 1160px) {
+					font-size: 40px;
+				}
+				@media (max-width : 990px) {
+					font-size: 35px;
+				}
+				@media (max-width : 480px) {
+					line-height: 1;
+				}
+			}
+			&-text {
+				@media (max-width : 480px) {
+					font-size: 12px;
+				}
+			}
+			&-btn {
+				width: 170px;
+				height: 50px;
+				text-align: center;
+				@media (max-width : 990px) {
+					margin: 20px auto;
+				}
+			}
+		}
+		&__img {
+		  flex: 1.3;
+		  height: 340px;
+		  background-repeat: no-repeat;
+		  background-position: center center;
+		  background-size: cover;
+		  @media (max-width : 990px) {
+		    flex: unset;
+		    height: 450px;
+		  }
+		  @media (max-width : 550px) {
+		    flex: unset;
+		    height: 250px;
+		  }
+		}
+	}
 }
 
-.room {
-	display: flex;
-	justify-content: space-between;
-	margin-bottom: 70px;
-	@media (max-width : 990px) {
-		flex-direction: column;
-		justify-content: center;
-	}
+//testimonials
+.testimonials {
+	padding-bottom: 75px;
+	text-align: center;
 	@media (max-width : 480px) {
-		margin-bottom: 35px;
+		padding-bottom: 40px;
+	}
+	&__title {
+		font-size: 50px;
+		font-family: AGaramondPro, sans-serif;
+	}
+	&__slider {
+		position: relative;	
+		width: 640px;
+		margin: 0 auto;
+		@media (max-width : 680px) {
+			max-width: 100%;
+		}
+	}
+	.testimonial__slider {
+		&-text {
+			font-size: 23px;
+			@media (max-width : 640px) {
+				font-size: 18px;
+			}
+			@media (max-width : 520px) {
+				font-size: 16px;
+			}
+		}
+		&-name {
+			@media (max-width : 520px) {
+				font-size: 14px;
+			}
+		}
 	}
-	&__info {
+	.slick {
+		&-slide {
+			outline: none;
+		}
+		&-arrow {
+			outline: none;
+			border: none;
+			width: 40px;
+			height: 40px;
+			background: #E0B973;
+			cursor: pointer;
+			position: absolute;
+			z-index: 5;
+			border-radius: 8px;
+			top: 95px;
+			@media (max-width : 480px) {
+				display: none;
```

Comment:
if you hide arrows on mobile how user can know that it's slider?

You need to add dots and hide arrows but not in css - you should use responsive property in slick slider
So, in you need at the beginning `arrow:true, dots:false` but then `arrow:false, dots:true`

<img width="747" alt="Screenshot 2021-01-27 at 09 11 01" src="https://user-images.githubusercontent.com/13525860/105955885-a62cd080-607f-11eb-9eed-4691d2190a12.png">